<?xml version="1.0" encoding="UTF-8"?>
<comment>
 /*
 * extractiondialog.ui
 *
 * Copyright (C) 2012 basysKom GmbH (info@basyskom.com)
 *
 * This library is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public
 * License as published by the Free Software Foundation; either
 * version 2.1 of the License, or (at your option) any later version.
 *
 * This library is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
 * Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public
 * License along with this library; if not, write to the Free Software
 * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
 *
 */
</comment>
<ui version="4.0">
 <class>ExtractionDialog</class>
 <widget class="QWidget" name="ExtractionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>427</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="KTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="accessibleName">
      <string>AdvancedOptionsTab</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="automaticResizeTabs">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="destinationTab">
      <attribute name="title">
       <string comment="@title:tab like in 'General options'">General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Destination path (will be created if it does not exist)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="optionsTab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="accessibleName">
       <string>AdvancedOptionsTab</string>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <attribute name="title">
       <string comment="@title:tab like in 'Advanced options'">Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Files and Paths</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Overwrite handling</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KComboBox" name="conflictsComboBox">
              <item>
               <property name="text">
                <string>Ask before overwrite</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Overwrite without asking</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Skip existing files</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Rename extracted files</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="autoSubfoldersCheckBox">
            <property name="toolTip">
             <string>Creates a subfolder for each archive</string>
            </property>
            <property name="text">
             <string>&amp;Automatically create subfolders</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="preservePathsCheckBox">
            <property name="toolTip">
             <string>Unpack to the absolute paths used in the archive</string>
            </property>
            <property name="text">
             <string>&amp;Preserve Paths</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="systemGroupBox">
         <property name="title">
          <string>System</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="multithreadingCheckBox">
              <property name="toolTip">
               <string>Use as many parallel threads as possible</string>
              </property>
              <property name="text">
               <string>Use &amp;Multithreading</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="utf8CheckBox">
              <property name="toolTip">
               <string>Fixes the filename encoding</string>
              </property>
              <property name="text">
               <string>&amp;Fix filename encoding</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="openFolderCheckBox">
              <property name="toolTip">
               <string>Opens the destination folder in a file browser</string>
              </property>
              <property name="text">
               <string>Open &amp;destination folder after extraction</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="closeAfterExtractionCheckBox">
              <property name="toolTip">
               <string>Ark will be close when all files have been extracted</string>
              </property>
              <property name="text">
               <string>&amp;Close Ark after extraction</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>43</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
